src/NewsService.Data.Provider.MsSql.Ef/Migrations/20210812164100_AddServicePropertiesToDb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using LT.DigitalOffice.NewsService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.NewsService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(NewsServiceDbContext))]
+    [Migration("20210812164100_AddServiceProperties")]
+    public class AddServiceProperties : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Guid>(
+                name: nameof(DbNews.CreatedBy),
+                table: "News",
+                nullable: false);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: nameof(DbNews.CreatedAtUtc),
+                table: "News",
+                nullable: false);
+
+            migrationBuilder.AddColumn<Guid>(
+                name: nameof(DbNews.ModifiedBy),
+                table: "News",
```

Comment:
use DbNew.TableName

Reply:
I looked at other migrations, we don't seem to have written like that anywhere